deployment/media-insights-stack.yaml

```diff
@@ -608,7 +601,7 @@ Resources:
 
   # Services - S3
   # TODO: Best Practice - S3 Bucket should have access logging configured, S3 Bucket should have encryption option set
-  DataplaneBucket:
+  Dataplane:
```

Comment:
I changed this for the case if we ever want to reference the logical id of this resource as the name for a URL - such as required for Kibana. Not needed now, but no issues by changing it preemptively.